test/e2e_node/cpu_manager_test.go

```diff
@@ -115,8 +117,13 @@ func waitForContainerRemoval(ctnPartName string) {
 }
 
 func isHTEnabled() bool {
-	err := exec.Command("/bin/sh", "-c", "if [[ $(lscpu | grep \"Thread(s) per core:\" | cut -c24) != \"2\" ]]; then exit 1; fi").Run()
-	if err != nil {
+	outData, err := exec.Command("/bin/sh", "-c", "lscpu | grep \"Thread(s) per core:\" | cut -d \":\" -f 2").Output()
+	framework.ExpectNoError(err)
+
+	threadsPerCore, err := strconv.Atoi(strings.TrimSpace(string(outData)))
+	framework.ExpectNoError(err)
+
+	if threadsPerCore != 2 {
```

Comment:
We should just return threadsPerCore > 1 instead right? The rest of the CPU manager code doesn't make assumptions about how many HW threads each physical core can support. This will be greater than two on some architectures.